src/main/resources/assets/structurize/lang/en_us.json

```diff
@@ -411,7 +411,7 @@
   "block.structurize.double_crossed_dark_oak_cobble_stone_timber_frame": "Double Crossed Dark Oak Cobblestone Timber Frame",
   "block.structurize.white_clay_acacia_shingle": "White Clay Acacia Shingle",
   "block.structurize.white_clay_oak_shingle": "White Clay Oak Shingle",
-  "structurize.config.maxoperationspertick": "Max Operations Per Tick",
+  "structurize.config.maxoperationspertick": "Max Operations per Tick:",
```

Comment:
Don't use colons in config names, they will be used in guis once Forge will make config gui a thing